proto/prysm/v1alpha1/attestation/id_test.go

```diff
@@ -50,14 +66,6 @@ func TestNewId(t *testing.T) {
 		cb := primitives.NewAttestationCommitteeBits()
 		att := util.HydrateAttestationElectra(&ethpb.AttestationElectra{CommitteeBits: cb})
 		_, err := attestation.NewId(att, attestation.Data)
-		assert.ErrorContains(t, "0 committee bits are set", err)
-	})
-	t.Run("data source Electra - multiple bits set", func(t *testing.T) {
-		cb := primitives.NewAttestationCommitteeBits()
-		cb.SetBitAt(0, true)
-		cb.SetBitAt(1, true)
-		att := util.HydrateAttestationElectra(&ethpb.AttestationElectra{CommitteeBits: cb})
-		_, err := attestation.NewId(att, attestation.Data)
-		assert.ErrorContains(t, "2 committee bits are set", err)
```

Comment:
Should we not keep this test?

Reply:
no because this asserts that setting 2 committee bits is wrong, while this PR allows it